### ISSUE TITLE
precompile for distribution

### ISSUE THREAD

**Issue by VinSpee** [idolize/redux-requests#5]
Hi!

I'd love to use this module, but it doesn't compile in a typical setup because it's written in es6 but not compiled when distributed. For example, importing it in a project gives the error

``` js
ERROR in ./~/redux-requests/index.js
Module parse failed: /Users/me/Projects/prototypes/myproject/node_modules/redux-requests/index.js Line 11: Unexpected token
You may need an appropriate loader to handle this file type.
|  * @return {Object}        The new state.
|  */
| export function requestsReducer(state = {}, action) {
|   if (!action.meta || !action.meta.httpRequest || !action.meta.httpRequest.url) {
|     return state;
```

redux and react router get around this with an npm `republish` script. I think it'd be a great idea to set one up here as well.

Thanks!

**Comment by idolize** [comment]
Hey @VinSpee,

Oops! Thanks for the heads up. I actually do have a [`prepublish` script](https://github.com/idolize/redux-requests/blob/master/package.json#L27) to compile the ES6 using Babel, but I was pointing `main` in `package.json` to the wrong location (ES6 source)!

**Comment by idolize** [comment]
Released on npm as `redux-requests@1.0.2`

**Comment by VinSpee** [comment]
aweomse, thanks!
